actions/global_actions.test.js

```diff
@@ -2,13 +2,32 @@
 // See LICENSE.txt for license information.
 
 import {browserHistory} from 'utils/browser_history';
+import {closeRightHandSide, closeMenu as closeRhsMenu} from 'actions/views/rhs';
+import {close as closeLhs} from 'actions/views/lhs';
 
-import {redirectUserToDefaultTeam} from 'actions/global_actions.jsx';
+import {redirectUserToDefaultTeam, toggleSideBarRightMenuAction} from 'actions/global_actions.jsx';
+
+jest.mock('actions/views/rhs', () => ({
+    closeMenu: jest.fn(),
+    closeRightHandSide: jest.fn(),
+}));
+
+jest.mock('actions/views/lhs', () => ({
+    close: jest.fn(),
+}));
 
 describe('actions/global_actions', () => {
     test('redirectUserToDefaultTeam', async () => {
         browserHistory.push = jest.fn();
         await redirectUserToDefaultTeam();
         expect(browserHistory.push).toHaveBeenCalledWith('/select_team');
     });
+
+    test('toggleSideBarRightMenuAction', () => {
+        const dispatchMock = () => {};
+        toggleSideBarRightMenuAction()(dispatchMock);
+        expect(closeRhsMenu).toHaveBeenCalled();
```

Comment:
I felt testing like this made more sense than asserting state on store. Assertion for store change felt it should be part of the individual action.

Reply:
Just an FYI: Test is not related to the PR